test/integration/api/assignments.js

```diff
@@ -499,7 +507,7 @@ describe('/projects/:id/assignments', () => {
     it('should assign the actor by role numeric id', testService((service) =>
       service.login('bob', (asBob) =>
         Promise.all([
-          service.get('/v1/roles/manager').expect(200).then(({ body }) => body.id),
+          asBob.get('/v1/roles/manager').expect(200).then(({ body }) => body.id),
```

Comment:
This endpoint does not currently require authentication.  How does that expectation changing relate to 401 vs 403 errors?

Reply:
Those two changes (more 401 errors + requiring auth for /v1/roles) both touch on the theme of anonymity. However, I agree that they're also conceptually distinct. If it's easy to pull the two changes apart, that'd make this PR a little smaller, which sounds nice. I'm also OK with reviewing both changes as part of this PR. I like the idea of having the /v1/roles endpoints check auth.